.travis.yml

```diff
@@ -118,49 +122,35 @@ install:
     # WX requires OSMesa (mesa on conda) which has typically been an
     # additional test environment. With llvm=3.3 the combination of
     # EGL and mesa causes segmentation faults. See issue #1401.
+    # conda-forge does not provide qt4, so we try defaults on darwin
     - if [ "${DEPS}" == "full" ]; then
-        conda install --yes pyopengl scipy numpy$NUMPY networkx;
-        pip install -q numpydoc PySDL2;
+        conda install --yes pyopengl networkx numpydoc pysdl2;
         if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
-          conda install --yes matplotlib jupyter pyqt=4 pillow decorator six scikit-image;
+          conda install --yes matplotlib jupyter pyqt=4 pillow decorator six scikit-image glfw;
           if [ "${PYTHON}" == "3.6" ]; then
-            pip install -q freetype-py husl pypng cassowary imageio;
+            conda install --yes freetype-py imageio;
+            pip install -q husl pypng cassowary;
             rm -rf ${SRC_DIR}/vispy/ext/_bundled;
           else
-            pip install -q mock;
+            conda install --yes mock;
           fi;
         else
-          conda install --yes matplotlib jupyter pyqt=4 scikit-image;
-          pip install -q mock;
+          conda install --yes -c defaults pyqt=4;
+          conda install --yes matplotlib jupyter scikit-image mock;
         fi;
       fi;
 
+     # Install OSMesa
+    - if [ "${TRAVIS_OS_NAME}" == "linux" ] && [ "${DEPS}" == "osmesa" ]; then
```

Comment:
Could/should this be changed to just checking for osmesa?

Reply:
Yes, would make sense, since osmesa is just on linux atm.